specs/driver_spec.rb

```diff
@@ -77,4 +78,45 @@
       driver.average_rating.must_equal 0
     end
   end
+
+  describe "total_revenue method" do
+    it "returns the total revenue of driver trips" do
+      # Arrange
+
+      trip_dispatcher = RideShare::TripDispatcher.new
+      # Act
+
+      driver = trip_dispatcher.find_driver(1)
```

Comment:
This test works, but it looks like you have the pieces mis-labeled. In general, the _Act_ step is always going to be calling the method you're interested in - in this case `driver.total_revenue`. Anything before that is all part of _Arrange_.